### PR TITLE
rebuild image and add kb-sdk patch

### PR DESCRIPTION
**Pull request by Xiangs18**

### CODE REVIEW

File: .github/workflows/kb_sdk_test.yaml [hunk 1]
```diff
@@ -56,9 +43,12 @@ jobs:
...
       run: |
-        sh $GITHUB_WORKSPACE/kb_sdk_actions/bin/kb-sdk test
-        bash <(curl -s https://codecov.io/bash)
+        sh scripts/GHA/kb-sdk test --verbose
+
+    # - name: Upload coverage to Codecov
+    #   uses: codecov/codecov-action@v3
+    #   with:
+    #     token: ${{ secrets.CODECOV_TOKEN }}
+    #     fail_ci_if_error: true
```
Comment (MrCreosote) [inline .github/workflows/kb_sdk_test.yaml]: it looks like coverage is set up for this module:

https://github.com/kbase/genome_annotation_api/blob/master/Makefile#L46-L58

Comment (Xiangs18) [reply .github/workflows/kb_sdk_test.yaml]: My last commit did not comment codecov and it failed to find coverage file though.

Comment (MrCreosote) [reply .github/workflows/kb_sdk_test.yaml]: Based on the makefile coverage is running; you'll need to figure out where the coverage file is ending up and how to get codecov to find it

Comment (Xiangs18) [reply .github/workflows/kb_sdk_test.yaml]: fixed

**Comment by Xiangs18** [comment]
Should coverage run against installed_clients files?

Screenshot 2024-07-29 at 5 23 59 PM

**Review by MrCreosote — APPROVED** [review]

**Comment by MrCreosote** [comment]
Did it run against the clients before? If so it's no worse

But no, it shouldn't, but it doesn't need to be fixed right now

**Comment by Xiangs18** [comment]
This is the first time Codecov run on this repo. I guess CI test as well.

**Comment by MrCreosote** [comment]
But was it configured to run against the clients?

**Pull request merged**